lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1710,36 +1710,30 @@ SwiftExpressionParser::Parse(DiagnosticManager &diagnostic_manager,
     return ParseResult::unrecoverable_error;
   }
 
-  if (log) {
+  auto dumpModule = [&](const char *msg) {
     std::string s;
     llvm::raw_string_ostream ss(s);
     parsed_expr->source_file.dump(ss);
     ss.flush();
+    LLDB_LOG(log, "\n{0}\n\n{1}\n", msg, s);
```

Comment:
Will LLDB_LOG format this? Shouldn't it be LLDB_LOGF?

Reply:
LLDB_LOGF -> printf
LLDB_LOG -> format